tests/integration/config-capture-introspection-queries.test.js

```diff
@@ -27,75 +31,91 @@ const queries = [
   }`
 ]
 
-setupApolloServerTests({
-  suiteName: 'default',
-  createTests: createCaptureIntrospectionTests.bind(null, true)
-})
+const defaultTests = generateTests(true)
+const captureTrueTests = generateTests(false)
+const captureFalseTests = generateTests(true)
 
-setupApolloServerTests({
-  suiteName: 'captureIntrospectionQueries: true',
-  createTests: createCaptureIntrospectionTests.bind(null, false),
-  pluginConfig: {
-    captureIntrospectionQueries: true
-  }
-})
+runTests({ tests: defaultTests })
+  .then(() =>
+    runTests({
+      tests: captureTrueTests,
+      suiteName: 'captureIntrospectionQueries: true',
+      pluginConfig: { captureIntrospectionQueries: true }
+    })
+  )
+  .then(() =>
+    runTests({
+      tests: captureFalseTests,
+      suiteName: 'captureIntrospectionQueries: false',
+      pluginConfig: { captureIntrospectionQueries: false }
+    })
+  )
```

Comment:
Attempting to remove this setup resulted in a muddied state and failing tests. Basically, an assertion for `true` was finding a `false` due to the `pluginConfig` not being scoped.